src/workerd/api/blob-test.capnp

```diff
@@ -0,0 +1,14 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "blob-test",
```

Comment:
do you think we can generate this file in most cases? Would be great to create just 1 file for test rather than two.

Reply:
I thought a lot about this. Many tests will need to customize the configuration, e.g. to test bindings, set up actors, or customize compat flags. I don't think we can avoid having some sort of way to specify configuration.

But I agree it would be nice to have the code and config in a single file. This is one of my future TODO points in the PR description.

One idea I've had is to introduce a new block-quote syntax in Cap'n Proto that is convenient for quoting large blocks of text, so that you could the JavaScript conveniently in the capnp file, like:

```
using Workerd = import "/workerd/workerd.capnp";

const unitTests :Workerd.Config = (
  services = [
    ( name = "blob-test",
      worker = (
        modules = [
          (name = "worker", esModule = .code)
        ],
        compatibilityDate = "2023-01-15",
      )
    ),
  ],
);

const code :Text = """
// JavaScript code here!
export default {
  async test(ctrl, env, ctx) {
    // ...
  }
}
"""
```

However, this might have some challenges when it comes to things like syntax highlighting?

Thoughts?

Reply:
You could solve the syntax highlighting challenges by allowing something like this in capnp:

```
const code :Text = js"""
// JavaScript code here!
export default {
  async test(ctrl, env, ctx) {
    // ...
  }
}
"""
```

Though I personally don't think having two separate files is a big deal.